client/my-sites/stats/activity-log-item/index.jsx

```diff
@@ -310,6 +313,7 @@ class ActivityLogItem extends Component {
 	renderSummary() {
 		const {
 			allowRestore,
```

Comment:
hmm, no, after testing the PR, decided that is not a good idea to have disabled rewind buttons everywhere if you can never rewind.